src/plugins/wb-jsonmanager/jsonmanager-fr.hbs

```diff
@@ -344,3 +344,288 @@
+<div data-wb-jsonmanager='{
+	"name": "ex1",
+	"extractor": [
+					{ "selector": "title", "path": "pageTitle" },
+					{ "selector": "h2", "path": "firstH2" },
+					{ "selector": "h4", "path": "allH4", "selectAll": true }
+				],
+	"wraproot": "rootArray"
+}'>
+	<p> Le titre de la page est : <span data-json-replace="#[ex1]/rootArray/pageTitle">Non disponible</span></p>
+	<p> Le premier H2 est : <span data-json-replace="#[ex1]/rootArray/firstH2">Non disponible</span></p>
+	<p>Tous les H4 de la page :</p>
+	<ul data-wb-json='{
+						"url": "#[ex1]/rootArray/allH4",
+						"mapping": [
+							{ "selector": "li" }
+						]
+					}'>
+		<template>
+			<li></li>
+		</template>
+	</ul>
+</div>
+
+<details>
+<summary>Afficher le code source</summary>
+<pre><code>
+&lt;p&gt;Utilisez &lt;code&gt;replace&lt;/code&gt; et &lt;code&gt;mapping&lt;/code&gt; sur la même source de données JSON&lt;/p&gt;
+
+&lt;div data-wb-jsonmanager='{
+		&quot;name&quot; : &quot;ex1&quot;,
+		&quot;extractor&quot; : [
+			{ &quot;selector&quot; : &quot;title&quot;, &quot;path&quot; : &quot;pageTitle&quot; },
+			{ &quot;selector&quot; : &quot;h2&quot;, &quot;path&quot; : &quot;firstH2&quot;},
+			{ &quot;selector&quot; : &quot;h4&quot;, &quot;path&quot; : &quot;allH4&quot;, &quot;selectAll&quot; : true}
+		],
+		&quot;wraproot&quot;: &quot;rootArray&quot;
+	}'>
+	&lt;p&gt;Le titre de la page est : &lt;span data-json-replace=&quot;#[ex1]/rootArray/pageTitle&quot;&gt;Non disponible&lt;/span&gt;&lt;/p&gt;
+	&lt;p&gt;Le premier H2 est : &lt;span data-json-replace="#[ex1]/rootArray/firstH2&quot;&gt;Non disponible&lt;/span&gt;&lt;/p&gt;
+	&lt;p&gt;Tous les H4 de la page :&lt;/p&gt;
+	&lt;ul data-wb-json='{
+			&quot;url&quot; : &quot;#[ex1]/rootArray/allH4&quot;,
+				&quot;mapping&quot; : [
+					{ &quot;selector&quot; : &quot;li&quot; }
+				]
+		}'>
+		&lt;template&gt;
+			&lt;li&gt;&lt;/li&gt;
+		&lt;/template&gt;
+	&lt;/ul&gt;
+&lt;/div&gt;</code></pre>
+</details>
+
+<h3>Regrouper les sélecteurs sur la même source de données JSON</h3>
+<p>La liste de description ci-dessous est la source des sélecteurs combinés :</p>
+<dl data-dl="test">
+	<dt data-td="data-title-1">Titre 1</dt>
+	<dd>Description 1</dd>
+
+	<dt data-td="data-title-2">Titre 2</dt>
+	<dd>Description 2</dd>
+
+	<dt>Titre 3</dt>
+	<dd>Description 3</dd>
+</dl>
+
+<div data-wb-jsonmanager='{
+		"name": "ex2",
+		"url": "demo/data-en.json",
+		"extractor": [
+						{ "selector": "title", "path": "pageTitle" },
+						{ "selector": "h2", "path": "firstH2" },
+						{ "selector": "h3", "path": "allH3", "selectAll":true },
+						{
+							"selector": "dl[data-dl]",
+							"path": "monGroup",
+							"extractor": [
+								{
+									"selector": "dt",
+									"path": "title",
+									"attr":"data-td"
+								},
+								{
+									"selector": "dd",
+									"path": "desc"
+								}
+
+							]
+						}
+					],
+		"wraproot": "rootArray"
```

Comment:
This config need to be removed in order to have the working example functional

```suggestion
					]
```

Reply:
fixed